src/applications/mhv/landing-page/components/CardLayout.jsx

```diff
@@ -22,6 +22,7 @@ const CardLayout = ({ data }) => {
           <div
             key={`col-${y}`}
             className="vads-l-col--12 medium-screen:vads-l-col mhv-u-grid-gap vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0"
+            data-testid="mhv-link-group"
```

Comment:
Since this code is being called within `.map`, multiple elements with the same `data-testid` attribute will be created within the page.

Reply:
That was the intent, however, I am seeing in the documentation that this should be an ID, so I'll need to figure out how to do make them unique, but still be able to generically walk up the elements.

Reply:
Fixed